src/components/UserMasterPanel.js

```diff
@@ -20,10 +19,10 @@ const styles = (theme) => ({
 });
 
 const UserMasterPanel = (props) => {
-  const { classes, edited, readOnly, onEditedChanged, modulesManager,  obligatory_user_fields, obligatory_eo_fields} = props;
+  const { classes, edited, readOnly, onEditedChanged, modulesManager, obligatory_user_fields, obligatory_eo_fields } =
```

Comment:
obligatory_user_fields, obligatory_eo_fields -> obligatoryUserFields, obligatoryEOFields 

(we should use cammel case not 'snake' one)

Reply:
Done.